azure-devops/native-build-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     fixedFlags: '--timeout=240;--shuffle'
     parallelismFlag: '-j$(testParallelism)'
     xmlOutputFlag: '--xunit-xml-output=$(${{ parameters.buildOutputLocationVar }})/test-results.xml'
-    shardFlags: '--num-shards=$(System.TotalJobsInPhase);--run-shard=$(System.JobPositionInPhase)'
+    shardFlags: '-vv;--num-shards=$(System.TotalJobsInPhase);--run-shard=$(System.JobPositionInPhase)'

```

Comment:
Is `-vv` necessary/useful, or is it a relic of development?

Reply:
I _think_ this one is useful.  Without this, CI failures tell you which tests failed, but give very little output from the failing test.  But it's hard for me to verify that since this is in the CI machinery.

Reply:
I'd say this is useful since upstream broke our ability to write nice output to the xml test results.